python-ecosys/urequests/urequests.py

```diff
@@ -105,6 +116,12 @@ def request(method, url, data=None, json=None, headers={}, stream=None):
     resp = Response(s)
     resp.status_code = status
     resp.reason = reason
+
+    if stream:
+        resp.raw.setblocking(False)
+    else:
+        resp.content
```

Comment:
I'm not sure it makes sense to pre-read the content here.  Some uses of this may not need the content and then it shouldn't waste time and memory retrieving it.

Reply:
I have not considered such use case but it makes total sense. There is no need to read the response like the original requests package does. I'll remove 9b5d30c.